platform/mbed_rtc_time.cpp

```diff
@@ -23,16 +23,53 @@
 static SingletonPtr<PlatformMutex> _mutex;
 
 #if DEVICE_RTC
+
 static void (*_rtc_init)(void) = rtc_init;
 static int (*_rtc_isenabled)(void) = rtc_isenabled;
 static time_t (*_rtc_read)(void) = rtc_read;
 static void (*_rtc_write)(time_t t) = rtc_write;
-#else
+
+#elif DEVICE_LOWPOWERTIMER
+
+#include "drivers/LowPowerTimer.h"
+
+static SingletonPtr<mbed::LowPowerTimer> _rtc_lp_timer;
+static uint64_t _rtc_lp_base;
```

Comment:
I see that _rtc_lp_base is getting written in the write function below, how is it used?

Reply:
`_rtc_lp_base` is used as a start time and then we add the actual timer value to it as timer starts from 0.